app/lib/sms-games/controllers/SGCreateFromMobileController.js

```diff
@@ -55,7 +55,6 @@ SGCreateFromMobileController.prototype.processRequest = function(request, respon
   }
 
   self = this;
-  self.createGame = createGame;
   request = request;
```

Comment:
Perhaps to access the request in the callback? Still seems to be working if we take this out, though.